src/rank_llm/retrieve_and_rerank.py

```diff
@@ -48,9 +54,13 @@ def retrieve_and_rerank(
             keys=openai_keys,
             **(get_azure_openai_args() if use_azure_openai else {}),
         )
-    elif "vicuna" in model_path.lower() or "zephyr" in model_path.lower():
+    elif model_path.lower()=="rank_zephyr" or model_path.lower()=="rank_vicuna":

```

Comment:
why this change? It will break the code if we have local checkpoints of finetuned models. 
Within the existing elif, you can handle rank_zephyr and rank_vicuna as special cases.